components/script/dom/htmltablerowelement.rs

```diff
@@ -54,6 +56,14 @@ impl HTMLTableRowElement {
                            document,
                            HTMLTableRowElementBinding::Wrap)
     }
+
+    /// Determine the index for this `HTMLTableRowElement` within the given
+    /// `HTMLCollection`. Returns `-1` if not found within collection.
+    fn row_index(&self, collection: Root<HTMLCollection>) -> i32 {
```

Comment:
```
8:49 AM <frewsxcv> KiChjang: i'm returning -1 because it gets used like four times in that file, otherwise i have .unwrap_or(-1) everywhere
8:50 AM <frewsxcv> i understand your argument, but it's also just an internal utility to that struct
8:50 AM <frewsxcv> i don't feel strongly though
8:50 AM <•Ms2ger> I'm with frewsxcv on this one
8:50 AM <KiChjang> ah, it's not public?
8:50 AM <KiChjang> then i'm fine with that
```